package.json

```diff
@@ -19,6 +19,6 @@
     "coffeelint": "^1.10.1"
   },
   "dependencies": {
-    "tree-sitter-json": "^0.13.1"
+    "tree-sitter-json": "git://github.com/jasonrudolph/tree-sitter-json.git#e83c1bcaa3379ce42be9d3a7c70b6fa8ff2c2e1d"
```

Comment:
Before merging this pull request, I'll need to get https://github.com/tree-sitter/tree-sitter-json/pull/8 merged, publish a new version of tree-sitter-json, and then update this dependency to reference that new version.